packages/frontend/app/(auth)/login/components/LoginRedirectInfoProvider.tsx

```diff
@@ -0,0 +1,71 @@
+'use client'
```

Comment:
Note: I renamed this component (used to be called OrganizationProviderForInvitedCourse) and remade it so you can pass down courseId,queueId,and courseInviteCode which would then be used to call endpoints to enroll in the course, but then i realised that this was unnecessary and I could just access all this cookie data on the backend. I did end up using the invitedQueueId to show a little "You have been invited to a queue, please login to continue" message